ext/simplexml/simplexml.c

```diff
@@ -485,7 +485,7 @@ static zval *sxe_prop_dim_write(zend_object *object, zval *member, zval *value,
 			 * and could also be E_PARSE, but we use this only during parsing
 			 * and this is during runtime.
 			 */
-			zend_throw_error(NULL, "Cannot create unnamed attribute");
+			zend_value_error("Cannot create or write attributes with an empty name");
```

Comment:
It might make sense to use a different error message for this (vs `['']`). Maybe something like `Cannot append to an attribute list` (if that makes sense in this context?)

Reply:
> if that makes sense in this context?

Based on the comment above + the original message... It seems so for me.